### DISCUSSION TITLE
Combobox: unable to customize loading and no-results templates

### DISCUSSION THREAD

**Discussion by gaddoz** [vmware/clarity#5905] [General]
I'm currently trying to figure out how to customize the messages or templates for "loading" and "no results".

When using the asynchronous combobox like in the documentation, elements with classes .clr-combobox-options-empty-text and .clr-combobox-options-loading are added but it looks like it's impossible to change the labels.

I've accomplished to change labels hiding .clr-combobox-options-empty-text and .clr-combobox-options-loading with a css rule, but I'm wondering if there is cleaner way or if it makes sense to create a feature-request.

**Comment by gnomeontherun** [comment]
@Jinnie

**Comment by Jinnie** [comment]
The customization currently available is based on the Internationalization API, as explained here:
https://v4.clarity.design/documentation/internationalization

using these keys:
- loading
- comboboxNoResults

Full template customization is not considered for Clarity Angular
